packages/hash/external-services/docker-compose.yml

```diff
@@ -43,14 +42,47 @@ services:
     environment:
       - PGDATA=/data/pgdata
     ports:
-      - 5432:5432
+      - "${POSTGRES_PORT}:5432"
     volumes:
       - ../../../var/external-services/postgres/data:/data
-
+      - ./postgres/postgresql.conf:/etc/postgresql/postgresql.conf:ro
+      - ./postgres/init-user-db.sh:/docker-entrypoint-initdb.d/init-user-db.sh
```

Comment:
See [this PR](https://github.com/hashintel/hash/pull/862) for the thinking behind this developer PG setup